modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -61,24 +63,37 @@ metadata:
 type: Opaque
 stringData:
  nmstate: |
-  routes:
-   config:
-   - destination: 0.0.0.0/0
-     next-hop-address: <next_hop_address>
-     next-hop-interface: <next_hop_interface>
-  dns-resolver:
-   config:
-    server:
-    - <dns_server_ip_address>
-  interfaces:
-  - name: <nic_interface>
-    state: up
-    ipv4:
-     address:
-     - ip: <ip_address>
-       prefix-length: <cidr>
-     enabled: true
-     dhcp: false
+  hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: redfish+http://<out_of_band_ip>/redfish/v1/Systems/
+          username: <user>
+          password: <password>
+          disableCertificateVerification: null
+        bootMACAddress: <NIC1_mac_address>
+        bootMode: UEFI
+        rootDeviceHints:
+          deviceName: "/dev/sda"
+        networkConfig: <3>
+          interfaces:
+          - name: <nic1_name> <4>
+            type: ethernet
+            state: up
+            ipv4:
+              address:
+              - ip: <ip_address> <4>
+                prefix-length: 24
+              enabled: true
+          dns-resolver:
+            config:
+              server:
+              - <dns_ip_address> <4>
+          routes:
+            config:
+            - destination: 0.0.0.0/0
+              next-hop-address: <next_hop_ip_address> <4>
+              next-hop-interface: <next_hop_nic1_name> <4>
```

Comment:
This YAML sample and the one that follows do not lint correctly for me on http://www.yamllint.com/. It looks like they have too many sections. If this is intentional, it's fine!

Reply:
yes, I agree, the lint is not ok.  on host section, the "- name" should aligned with hosts, meaning same indentation

Reply:
Here is an example with correct indentation:

hosts:
- name: openshift-master-0
  role: master
  bmc:
    address: redfish+http://<out_of_band_ip>/redfish/v1/Systems/
    username: <user>
    password: <password>
    disableCertificateVerification: null
  bootMACAddress: <NIC1_mac_address>
  bootMode: UEFI
  rootDeviceHints:
    deviceName: “/dev/sda”
  networkConfig:
    interfaces:
    - name: <nic1_name> <4>
      type: ethernet
      state: up
      ipv4:
        address:
        - ip: <ip_address> <4>
          prefix-length: 24
        enabled: true
    dns-resolver:
      config:
        server:
        - <dns_ip_address> <4>
    routes:
      config:
      - destination: 0.0.0.0/0
        next-hop-address: <next_hop_ip_address> <4>
        next-hop-interface: <next_hop_nic1_name> <4>